src/tests/dbus-tests/test_90_swapspace.py

```diff
@@ -0,0 +1,86 @@
+#!/bin/python3
+
+import unittest
+import storagedtestcase
+import subprocess
+import time
+import dbus
+import glob
+import os
+
+class StoragedSwapSpaceTest(storagedtestcase.StoragedTestCase):
+    """This is SwapSpace related functions unit test"""
+
+    def get_device(self, dev_name):
+        dev = self.get_object('', '/block_devices/' + os.path.basename(dev_name))
+        return dev
+
+
+    def get_swapspace_iface(self, device):
+        iface = dbus.Interface(device, dbus_interface=self.iface_prefix + '.Swapspace')
+        return iface
+
+
+    def setUp(self):
+        # create new fake swap device
+        subprocess.call(['modprobe', 'scsi_debug'])
```

Comment:
Do you really need scsi_debug devices here? targetcli devices (`self.vdevs`) should work here.